app/views/food_recipes/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<p style="color: green"><%= notice %></p>
+
+<section class="shopping_list">
+<h1 class="shopping_list_text">Shopping List</h1>
+<div class="things_to_buy">
+<p>Amount of Food Items to buy: <%= @food_count%></p>
+<p>Total Vaue of Food Needed: $<%= @food_cost%></p>
+</div>
+<div class="grid-container3">
+    <div class="grid-item3"><strong>Food</strong></div>
+    <div class="grid-item3"><strong>Unit Price</strong></div>
+    <div class="grid-item3"><strong>Quantity</strong></div>
+    <% if @foods.empty?%>
+    <% else %>
+    <% @foods.each do |food| %>
+    <div class="grid-item3"><%= food.name%></div>
+    <div class="grid-item3">$<%= food.price%></div>
+    <div class="grid-item3"><%= food.quantity%><%=food.measurement_unit%></div>
```

Comment:
- [ Required ] I think it would be better to style the `general_shopping` list the same way it has been styled in the project wireframe. If you look at the project wireframe it is zebra stripped, please see the screenshot below for more details on this.

## Project wireframe
![Screenshot from 2023-07-01 14-01-09](https://github.com/calebchris000/Recipe-App/assets/16392046/cdd36c19-5441-41c4-baef-1b676eb36dc2)